conf/core-site.xml

```diff
@@ -0,0 +1,6 @@
+<configuration>
+  <property>
+      <name>fs.s3.impl</name>
```

Comment:
mapping `s3a` also

Reply:
no need to map s3a, s3a is already builtin mapped. 

https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html